examples/covtype.py

```diff
@@ -30,25 +34,74 @@ def _load_dataset():
     print("Data shape:", features.shape)
     print("Label distribution: {} has label 1, {} has label 0"
           .format(labels.sum(), N - labels.sum()))
-    return features, labels
+    return features[::5], labels[::5]
```

Comment:
I think we can use the whole dataset for the current algorithm.

Reply:
Agree.